aws-resourceexplorer2-defaultviewassociation/src/test/java/software/amazon/resourceexplorer2/defaultviewassociation/CreateHandlerTest.java

```diff
@@ -82,13 +82,15 @@ public void handleRequest_NonExistedDefaultView_Success() {
                 .awsAccountId(ACCOUNT_ID)
                 .build();
 
+        CallbackContext context = new CallbackContext();
+
         final ProgressEvent<ResourceModel, CallbackContext> response
-                = handler.handleRequest(proxy, request, null, logger);
+                = handler.handleRequest(proxy, request, context, logger);
 
         assertThat(response).isNotNull();
-        assertThat(response.getStatus()).isEqualTo(OperationStatus.SUCCESS);
-        assertThat(response.getCallbackContext()).isNull();
-        assertThat(response.getCallbackDelaySeconds()).isEqualTo(0);
+        assertThat(response.getStatus()).isEqualTo(OperationStatus.IN_PROGRESS);
```

Comment:
Should we rename this test method now that we don't expect it to succeed anymore?